app/room.py

```diff
@@ -0,0 +1,129 @@
+from abc import abstractmethod
+from person import Fellow, Staff
+
+
+class Room(object):
+    """docstring for Room."""
```

Comment:
This is not how you write a function doc string. Please put a better description than what is there.